databricks_cli/configure/cli.py

```diff
@@ -47,10 +55,19 @@ def configure_cli():
                             confirmation_prompt=True)
     if password == default_password:
         password = conf.password
-    config = DatabricksConfig(host, username, password)
+    config = DatabricksConfig.construct_from_password(host, username, password)
     config.overwrite()
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--token', show_default=True, is_flag=True, default=False)
```

Comment:
how could user know there is an option to use token?

Reply:
Let's document it better in `README.rst`. I added documentation pointing users to the two ways to configure the CLI for authentication.